Allan-breakout-room/Evan Hameed/style.css

```
@@ -0,0 +1,132 @@
+*{
+
+    padding: 0;
+    margin: 0;
+    box-sizing: border-box;
+    font-family: sans-serif;
+    color: rgb(61, 60, 60);
+}
+
+.header{
+    height: 8vh;
+    display: flex;
+    justify-content: space-evenly;
+    align-items: center;
+    background-color: rgb(73,160,236);
+    box-shadow:0px 5px 15px #888888  ;
+}
+.nav-link{
+ color: white;
+ padding: 10px;
+ margin: 7px;
+ border-radius: 20px;
+
+ text-decoration: none;
+ font-size: 0.9em;
+ font-weight: bold;
+
+
+}
+ .white-link{
+
+    background-color: white;
+    color: rgb(73,160,236);
+    border: 2px solid rgb(73, 160, 236);
+ }
```

try always to pay attention for the  code  indentation and format
try to put new line between each two code sections and don't leave more than one there and don't leave new lines in random places in your code in order to make it more clean
```suggestion
 border-radius: 20px;
 text-decoration: none;
 font-size: 0.9em;
 font-weight: bold;
}

 .white-link{
    background-color: white;
    color: rgb(73,160,236);
    border: 2px solid rgb(73, 160, 236);
 }
```

---

Thank you a lot Mohammed for this feedback! I will definitely consider your suggestions!